aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -89,7 +89,7 @@ In the sample app, the `IMyDependency` service is registered with the concrete t
 [!code-csharp[](dependency-injection/samples/2.x/DependencyInjectionSample/Startup.cs?name=snippet1&highlight=5)]
 
 > [!NOTE]
-> Each `services.Add{SERVICE_NAME}` extension method adds (and potentially configures) services. For example, `services.AddMvc()` adds the services Razor Pages and MVC require. We recommended that apps follow this convention. Place extension methods in the <xref:Microsoft.Extensions.DependencyInjection?displayProperty=fullName> namespace to encapsulate groups of service registrations.
+> Each `services.Add{SERVICE_NAME}` extension method adds (and potentially configures) services. For example, `services.AddMvc()` adds the services Razor Pages and MVC require. We recommended that apps follow this convention. Place extension methods in the [Microsoft.Extensions.DependencyInjection](/dotnet/api/microsoft.extensions.dependencyinjection) namespace to encapsulate groups of service registrations.
```

Comment:
Hi @v-thepet ... Could you change this to ....

```
<xref:Microsoft.Extensions.DependencyInjection>
```

Reply:
@guardrex - ok I will try it again. I thought that xref was broken and changed it to Docs links throughout, per @tdykstra approval.

Reply:
Was weird ... those `displayProperty` ones are *supposed to work*. Anywho ... just removing that should do the trick. 🤞

Reply:
Actually, best to do this ....

```suggestion
> Each `services.Add{SERVICE_NAME}` extension method adds (and potentially configures) services. For example, `services.AddMvc()` adds the services Razor Pages and MVC require. We recommended that apps follow this convention. Place extension methods in the [Microsoft.Extensions.DependencyInjection](xref:Microsoft.Extensions.DependencyInjection) namespace to encapsulate groups of service registrations.
```

Reply:
I changed it on both ... Idk if it will do the full namespace. It might. This way tho it's safe. It will definitely show the full namespace and take advantage of the xref link.

Reply:
@v-thepet , per your email, you found one xref link that wasn't working:

> Most of the broken internal links are \<xref>s pointing to: 
<xref:Microsoft.Extensions.DependencyInjection.EntityFrameworkServiceCollectionExtensions.AddDbContext*>
> This xref doesn’t work

That one is still OK to use the docs link for, if the xref doesn't work.